code/game/objects/effects/anomalies.dm

```diff
@@ -229,7 +239,7 @@
 
 			var/y_distance = turf_to.y - turf_from.y
 			var/x_distance = turf_to.x - turf_from.x
-			for(var/atom/movable/A in urange(12, turf_from)) // iterate thru list of mobs in the area
+			for(var/atom/movable/A in urange(16, turf_from)) // iterate thru list of mobs in the area
```

Comment:
magic numbers are bad. make this a define at the top of the file and undef it at the bottom

Reply:
Done!